containers.go

```diff
@@ -825,3 +825,24 @@ func imageId(repo, tag string) (string, error) {
 
 	return images[0].ID, nil
 }
+
+// docker0 returns the standard Docker network interface ("docker0" as reported
+// by ifconfig or brctl) and whether it exists on the host. This interface
+// serves as a network bridge between Docker containers.
+//
+// The interface will not be returned if it does not exist. This can happen if
+// Docker is not installed; or if the host is running macOS or WSL2, as in these
+// operating systems the bridge exists inside the VM and not on the host.
+func docker0(ctx context.Context) (net.Interface, bool) {
```

Comment:
Since we're not using the interface directly, I'd rather name the function purpose to be more evocative of what we're using it for. I suggest `hostHasDockerNetwork` and drop the `net.Interface` return.

Further, the inability to list the network interfaces is a serious enough error that I think it should be surfaced as fatal, not a warning, so we can drop the `context.Context` parameter.

Everything together:

```suggestion
func hostHasDockerNetwork() (bool, error) {
```